test/Libraries/Microsoft.Extensions.AsyncState.Tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using Xunit;
```

Comment:
error IDE0073: A source file is missing a required header.

Reply:
Resolved